less/home-page.less

```diff
@@ -35,13 +67,26 @@
     .img-content {
       padding-right: 1%;
       padding-left: 0;
+
+      @media @mobile {
+        width: 100%;
+      }
+    }
+
+    img {
+      @media @mobile {
+        width: 100%;
+      }
```

Comment:
rather than repeating width : 100% a bunch of times, you could do one joint selector where you select all the elements you need to format to mobile with width : 100% , and then set it in one go.  example : 

.image-content , img, .inverse-content {
  @media @mobile {
          width : 100%;
   }
}